Expose Grafana setting `viewers_can_edit` to helm chart (no ticket)

erdii opened the pull request:
* add grafana.provisioning.configuration.viewers_can_edit to grafana's values.yaml
* change grafana.ini to expose flag setting to grafana

**What this PR does / why we need it**: Let `Viewers` explore loki logs without removing the source-of-truth for grafana dashboard configuration.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Documentation**:
<!-- Add links to the related documentation changes related to this pull request. E.g. the link to the kubermatic/docs pullrequest. -->

**Does this PR introduce a user-facing change?**:
<!--  Write your release note:
1. Enter your extended release note in the below block. If the PR requires additional action from users switching to the new release, include the string "action required".
2. If  no release note is required, just write "NONE".
-->
```release-note
NONE
```

erdii committed:
Expose Grafana setting `viewers_can_edit` to helm chart (no ticket)

* add grafana.provisioning.configuration.viewers_can_edit to grafana's values.yaml
* change grafana.ini to expose flag setting to grafana

erdii committed:
Upgrade forgotten minor version of grafana chart

erdii commented:
/retest

erdii commented:
/assign @xrstf

xrstf commented on config/monitoring/grafana/config/grafana.ini:
```diff
@@ -14,7 +14,8 @@ header_name = X-Auth-Username
+{{- with .Values.grafana.provisioning.configuration.auto_assign_org_role }}
 auto_assign_org_role = {{ . }}
-{{ end }}
+{{- end }}
```
Would you be so kind to improve this to just read

```
auto_assign_org_role = {{ .Values.grafana.provisioning.configuration.auto_assign_org_role }}
```

and remove the with...end statement? The value is always defined and now that we always have a `[user]` section, no need to be fancy.

xrstf reviewed (commented):
Suggestion

erdii committed:
Incorporate feedback from @xrstf

- remove with...end statement because users section will be always be there anyways.

erdii commented:
@xrstf incorporated your feedback :muscle:

xrstf commented:
Lovely

/lgtm
/approve
